src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
void should be`|null` to be completely correct. But I guess this nitpick is not too important.